scripts/start.js

```diff
@@ -58,13 +60,25 @@ const devServer = new WebpackDevServer(compiler, {
   quiet: true
 });
 
-// Launch WebpackDevServer.
-devServer.listen(port, function (err) {
+detect(DEFAULT_PORT, function (err, unoccupiedPort) {
+
   if (err) {
-    return console.log(err);
+    console.log(err);
   }
-});
 
-openBrowser('http://localhost:' + port + '/');
+  // TODO: add a prompt, so this message is flushed and user is aware of the port change.
+  if (DEFAULT_PORT !== unoccupiedPort) {
+    console.log(chalk.yellow('Default DEFAULT_PORT ' + DEFAULT_PORT + ' is already busy, we will use ' + unoccupiedPort));
+  }
+
+  // Launch WebpackDevServer.
+  devServer.listen(unoccupiedPort, function (err) {
+    if (err) {
+      return console.log(err);
+    }
+  });
+
+  openBrowser('http://localhost:' + unoccupiedPort + '/');
```

Comment:
Good catch it's a callback, which is getting executed, when server is running, I have updated the code a little bit.